target/linux/mediatek/filogic/base-files/etc/board.d/01_leds

```diff
@@ -187,6 +187,10 @@ smartrg,sdg-8733a)
 	ucidef_set_led_netdev "wan-orange" "WAN" "mdio-bus:08:orange:wan" "wan" "link_100 link_1000"
 	ucidef_set_led_netdev "wan-white" "WAN" "mdio-bus:08:white:wan" "wan" "link_10000"
 	;;
+teltonika,rutc50)
+	ucidef_set_led_netdev "wifi-2g" "wifi-2.4G" "green:wlan-2ghz" "phy0-ap0" "link tx rx"
+	ucidef_set_led_netdev "wifi-5g" "wifi-5G" "green:wlan-5ghz" "phy1-ap0" "link tx rx"
+	;;
```

Comment:
This should not be needed with the `linux,default-trigger = "phy0tpt";` entry in the device tree.